db/migrate/20221201201254_remove_remember_created_at_from_teachers.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveRememberCreatedAtFromTeachers < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :teachers, :remember_created_at, :string
```

Comment:
Can you please explain the reasoning behind this change? 😅 I can't grasp it, will deleting it won't cause any harm inside devise world? 😅